docs/sdks/server-sdks/node.md

```diff
@@ -156,3 +192,27 @@ We recommend always handling the `null` case in your code. Here are some example
 :::note
 It may take up to 10 seconds for changes to Eppo experiments to be reflected by the SDK assignments.
```

Comment:
does this feel a bit out of place?

Reply:
maybe we drop it, we already give that heads up in an in-app modal

Reply:
dropped it